vm/src/vm.rs

```diff
@@ -620,20 +620,28 @@ impl VirtualMachine {
         })
     }
 
-    pub fn _lt(&mut self, a: &PyObjectRef, b: PyObjectRef) -> PyResult {
-        self.call_method(a, "__lt__", vec![b])
+    pub fn _lt(&mut self, a: PyObjectRef, b: PyObjectRef) -> PyResult {
+        self.call_or_unsupported(a, b, "__lt__", "__lt__", |vm, a, b| {
```

Comment:
The reverse methods for these shouldn't be the same, i.e. they're not commutative like `__eq__` and `__ne__`; a > b != b > a. Instead, as described [here](https://docs.python.org/3/reference/datamodel.html):

>  __lt__() and __gt__() are each other’s reflection, __le__() and __ge__() are each other’s reflection

Reply:
Fixed